### ISSUE TITLE
wrong correction

### ISSUE THREAD

**Issue by kaushal2161** [ProjetPP/PPP-Spell-Checker#11]
I am parsing the question : what is area of Squircle? 
Squircle is there in wikidata but by parsing the question in Spell-checker module it gives ,
what is area of squiggle ? which is not in wikidata.
please have a look in this issue.

Waiting for your  response..

**Comment by Tpt** [comment]
Yes, Aspell, the library we rely on for spell checking, is not amazing. I'm currently working on a Wikidata lookup system that is lenient with spelling mistakes and should allow us to get ride of it.

**Comment by kaushal2161** [comment]
Thanks.

**Comment by progval** [comment]
Both the original input and the corrected one are sent to the Wikidata module by the core, so it should not be an issue.

**Comment by kaushal2161** [comment]
@ProgVal : i couldn't use core module because i have some issue with that. Details are available in issue#70 ProjetPP/Documentation module. May be you can again have a look in that.

**Comment by progval** [comment]
Just use the core without CoreNLP

**Comment by kaushal2161** [comment]
@ProgVal @Tpt Hi there.
I made a module for PPP and i have sent an email regarding this on 30th September but still waiting for reply.
Kindly give any feedback regarding email.

**Comment by progval** [comment]
We did not receive it. What was the address?

**Comment by progval** [comment]
Ok. Actually it got redirected from contact@ to our mailing list, but was not sent on the mailing list.

@eddycaron Could you approve the mail?

**Comment by kaushal2161** [comment]
Ok.
Once you get the mail just ping me for next steps.

Regards,
Kaushal.

**Comment by kaushal2161** [comment]
@ProgVal : Still didn't get the mail yet?
If not give some another email address where i can contact you

**Comment by progval** [comment]
> @ProgVal : Still didn't get the mail yet?

Nope :(

> If not give some another email address where i can contact you

firstname . lastname @ens-lyon.org

**Comment by kaushal2161** [comment]
Sorry, but is this email- address or i hv to make my own?

**Comment by progval** [comment]
Replace “firstname” by my first name and “lastname” by my last name, both available on my profile.
I do this to avoid spam bots.

**Comment by kaushal2161** [comment]
Ok great.

**Comment by kaushal2161** [comment]
Now, have you got the mail?

**Comment by progval** [comment]
Yes, I answered
